debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/MongoDbConnectorConfig.java

```diff
@@ -115,6 +115,7 @@ public static SnapshotMode parse(String value, String defaultValue) {
     public static final Field HOSTS = Field.create("mongodb.hosts")
             .withDisplayName("Hosts")
             .withType(Type.LIST)
+            .withGroup(Field.createGroupEntry(Field.Group.CONNECTION, 1))
```

Comment:
So why do we need this in addition to the existing grouping mechanism?

Reply:
It reflects the groups for the Debezium UI. Later we might use these as new groups.

Reply:
Yes, my question is: why don't we simply re-group the properties as per what's used for Debezium UI? Do you consider this to be too disruptive at this point?

Reply:
Exactly. I thought we had this discussion before and decided to first introduce it like that and later switch these groups to be used for the Kafka Connect groups.

Reply:
Ok, that works for me.